app/forms/idv/inherited_proofing/base_form.rb

```diff
@@ -43,12 +45,10 @@ def initialize(payload_hash:)
       end
 
       def submit
-        validate
-
         FormResponse.new(
-          success: valid?,
+          success: validate,
```

Comment:
Yes, either or I guess. I removed a previous call to `validate` at the top of the `#submit` method, then was calling `#valid?` a second time where it was unnesassary.

Reply:
In a lot of other spots in the app, to avoid calling `valid?` twice, we assign it to variable called `success`:

```
identity-idp:main> git grep "= valid?"
app/forms/delete_user_email_form.rb:    success = valid? && email_address_destroyed
app/forms/edit_phone_form.rb:    success = valid?
app/forms/event_disavowal/password_reset_from_disavowal_form.rb:      success = valid?
```

Reply:
Yes, that was an oversight on my part. I actually forgot that `#valid?` was an alias for `#validate`.

Reply:
`#valid?` is much more common that `#validate` in our codebase, I'd do this for consistency

```suggestion
          success: valid?,
```

Reply:
Addressed.